src/pages/User/components/UserHeader/utils/handleAvatarUpload.ts

```diff
@@ -15,10 +19,49 @@ export const handleAvatarUpload = async (file: File | null | undefined) => {
       .catch((err) => console.error(err));
 
     const user = LocalStorage.getItem('user') as User;
```

Comment:
Please use the util "getUser" 

https://github.com/Azordev/did-app/blob/14385ef549ac8efc0d8a74f76b5b181b740135c6/src/utils/auth.ts#L5-L20